### DISCUSSION TITLE
Option to hide ranking filters from the image browser

### DISCUSSION THREAD

**Discussion by younyokel** [Klace/stable-diffusion-webui-images-browser#3] [Ideas]
![image](https://user-images.githubusercontent.com/53343081/215548839-c7173a76-e794-4fe4-9e6f-71d1a66b454e.png)

I used to have Aesthetic Scorer installed, but now I don't, so these fields are redundant for me. Any way to remove them?

**Comment by Klace** [comment]
The ranking filters are actually for a custom ranking, not Aesthetic Scores. I can definitely make them optional though to clean up the UI!
